packages/db/src/core/schemas.ts

```diff
@@ -4,7 +4,6 @@ import { type ZodTypeDef, z } from 'zod';
 import { SERIALIZED_SQL_KEY, type SerializedSQL } from '../runtime/types.js';
 import { errorMap } from './integration/error-map.js';
 import type { NumberColumn, TextColumn } from './types.js';
-import { mapObject } from './utils.js';
```

Comment:
`core/utils` imports Vite so we want to avoid importing it — moved `mapObject` inline into this file as it is only used here